libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -843,3 +843,16 @@ void IotMqtt_ReceiveCallback( IotNetworkConnection_t pNetworkConnection,
 }
 
 /*-----------------------------------------------------------*/
+
+void IotMqtt_CloseCallback( IotNetworkConnection_t pNetworkConnection,
+                            IotNetworkCloseReason_t reason,
```

Comment:
Doesn't look like these parameters are used. Should be voided to get rid of unused parameter warnings.

Reply:
Good catch, fixed.